pp/go/cppbridge/head.go

```diff
@@ -177,7 +230,7 @@ func getSeriesIDFromBytes(data []byte) uint32 {
 }
 
 type HeadDataStorageSerializedChunks struct {
-	data []byte
+	*CBytes
```

Comment:
Here you add public property to the structure. I don't think that it is necessary. May be add constructor of this structure which will add finalizer and using pointer to private property with `[]byte` type is better solution. What you think about it?